primer/brand
# remove menu button from mobile viewport when no links passed

## Summary

Fixes incorrect rendering of the menu toggle button on narrow viewports, when no links are passed.

Fixes https://github.com/github/galaxy.github.com/issues/55

## List of notable changes:



- **added** guard for verifying more than 1 link is present
- **fixed** rendering loop in storybook story for conditional links

## What should reviewers focus on?

- Check out the story and verify button doesn't appear

## Steps to test:



1. Go to this story 
1. Check menu button isn't there

## Supporting resources (related issues, external links, etc):

- https://github.com/github/galaxy.github.com/issues/55

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2023-01-17 at 12 15 50](https://user-images.githubusercontent.com/13340707/212896745-b158d8f2-d86c-4358-9621-fa8865c8c0ab.png)


 </td>
<td valign="top">

![Screenshot 2023-01-17 at 12 15 32](https://user-images.githubusercontent.com/13340707/212896754-63ef128b-bf45-497c-b6bf-5e99cf4f714a.png)


</td>
</tr>
</table>
